test_package_docs/anonymous_library/anonymous_library-library.html

```diff
@@ -4,7 +4,7 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <meta name="generator" content="made with love by dartdoc 0.8.3-dev">
+  <meta name="generator" content="made with love by dartdoc 0.8.3">
```

Comment:
I think we should remove it from the leaf node files. Perhaps from index.html too, just for uniformity?